citylearn/electric_vehicle_charger.py

```diff
@@ -0,0 +1,374 @@
+import inspect
+from typing import List, Dict
+from citylearn.base import Environment
+
+from citylearn.electric_vehicle import electric_vehicle
+
+ZERO_DIVISION_CAPACITY = 0.00001
```

Comment:
This value is already defined in the `citylearn.energy_model` module so import it from there instead.